symphony-bdk-extensions/symphony-group-extension/src/main/java/com/symphony/bdk/ext/group/SymphonyGroupService.java

```diff
@@ -49,53 +56,122 @@ public SymphonyGroupService(RetryWithRecoveryBuilder<?> retryBuilder, ApiClientF
     this.typeApi = new TypeApi(client);
   }
 
+  /**
+   * Retrieve a specific type
+   *
+   * @param typeId the ID of the type to be retrieved
+   * @return the type details
+   */
   public Type getType(@Nonnull String typeId) {
     return this.executeAndRetry("groupExt.listTypes",
         () -> this.typeApi.getType("", typeId)
     );
   }
 
+  /**
+   * List all types
+   *
+   * @param status the status of the types to be returned (active or deleted)
+   * @param before NOT SUPPORTED YET, currently ignored
+   * @param after cursor that points to the end of the current page of data. If not present, the current page is the first page
+   * @param limit maximum number of items to return
+   * @param sortOrder items sorting direction (ordered by createdDate)
+   * @return the list of all matching types
+   */
   public TypeList listTypes(@Nullable Status status, @Nullable String before, @Nullable String after,
```

Comment:
_Type-related_ endpoints are not documented, maybe we should not expose them in BDK. Also, @symphony-enrico told me that the default type is `SDL`, other ones should not be used by public API